### PR TITLE
Switch landing page 'Log in' from Text to Button

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

The initial landing page "Log in" link is hard to press because it's text with a small hitbox:

screenshot from 2017-05-11 13-07-42

But looking at Zeplin, it's not supposed to be text on mobile, it's supposed to be a button.  (It's text on desktop.)

### CODE REVIEW

**Comment by cjb** [comment]
(also added an unrelated change to remove [in beta] from devices add)

**Comment by cjb** [comment]
also, start 1.0.2, fixes our failing iOS build

**Comment by cjb** [comment]
@MarcoPolo r?

**Comment by chrisnojima** [comment]
Should make a ticket to remove the feature flag for mobile

File: shared/devices/container.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const mapDispatchToProps = (dispatch: any, {routeState, setRouteState, navigateU
 })
 
 const menuItems = props => ([
-  ...flags.mobileAppsExist && [{onClick: props.addNewPhone, title: 'New phone [in beta]'}] || [],
+  ...flags.mobileAppsExist && [{onClick: props.addNewPhone, title: 'New phone'}] || [],
```
Comment (MarcoPolo) [inline shared/devices/container.js]: aww we can't be like gmail and be in beta forever?

**Pull request merged**